src/site/site.service.ts

```diff
@@ -87,12 +95,11 @@ export class SiteService {
             };
 
             await this.accountService.createInternalAccount(
-                newSite,
+                site,
                 internalAccountData,
-                tx,
```

Comment:
Have you already removed `tx` usage from `accountService.createInternalAccount`?